src/store/slices/auth.slice.ts

```diff
@@ -1,15 +1,16 @@
-import {createSlice, PayloadAction,} from "@reduxjs/toolkit";
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
 
 const initialState: boolean = false;
 
 export const authSlice = createSlice({
     name: 'auth',
     initialState,
     reducers: {
-        isLogged: (state, {payload}:PayloadAction<boolean>) => {
+        isLogged: (state, { payload }: PayloadAction<boolean>) => {
```

Comment:
not a suitable name for reducer as it makes some changes in store, not return value. It should be named like setLoginStaus